app/tasks/doctor.php

```diff
@@ -135,8 +139,15 @@
 
         try {
             $mail = $register->get('smtp'); /* @var $mail \PHPMailer\PHPMailer\PHPMailer */
+            if (empty($receiver)) {
```

Comment:
Hmm, looks like you used empty but the condition is opposite now. 
Let's change it to 

```suggestion
            if (!empty($receiver)) {
```